amr-wind/boundary_conditions/wall_models/WallFunction.cpp

```diff
@@ -138,14 +166,14 @@ void VelWallFunc::wall_model(
                         const amrex::Real vv =
                             vold_arr(i, j, k - 1 - idx_offset, 1);
                         const amrex::Real wspd = std::sqrt(uu * uu + vv * vv);
-                        // Dirichlet BC
+			// Dirichlet BC
                         varr(i, j, k, 2) = 0.0;
 
                         // Shear stress BC
                         varr(i, j, k, 0) =
-                            -tau.get_shear(uu, wspd) / mu * den(i, j, k);
+                            -tau.get_shear(uu, wspd, 0, 0, 0, 0) / mu * den(i, j, k);
```

Comment:
hmmmm... I wonder if we don't want to make default arguments. So the `get_shear` would look like:
```
get_shear(amrex::Real u, amrex::Real wspd, amrex::Real u_dx=0, amrex::Real v_dx=0, amrex::Real x_c=0, amrex::Real unit_nor=0)
```
and then we don't have to specify these extra arguments. It's a hack around an API that needs to be updated to something better. Thoughts anyone?